test/libsolidity/syntaxTests/enums/literal_conversion.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    enum Test { One, Two }
+    function f() public pure {
+        Test a = Test(0);
+        Test b = Test(1);
+        Test c = Test(type(uint).max);
+		a; b; c;
+    }
+}
+// ----
```

Comment:
Added positive test.

Reply:
Why the strange indentation? :)

Reply:
Or is it some leftover from debugging?

Reply:
Nah, it's mixing tabs and spaces. I'll fix it. The statements are there just to avoid the warning about unused variables.

Reply:
Fixed.

Reply:
You could just remove the variables. Only the cast is relevant anyway.

Reply:
Yeah, I like the positive test to produce no warnings or errors. So I prefer to leave the `a; b; c;` part.

Reply:
I meant removing `Type a = ` part and leaving just `Type(0);`. That would not produce warnings.

Anyway, that's just a suggestion. It's not like it makes all that much difference so leaving the variables is fine with me too.